delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/ProjectsDummy.scala

```diff
@@ -130,6 +130,8 @@ final class ProjectsDummy private (
 
   override def currentEvents(offset: Offset): fs2.Stream[Task, Envelope[ProjectEvent]] =
     journal.currentEvents(offset)
+
+  override def projectExists(project: ProjectRef): UIO[Boolean] = cache.fetch(project).map(_.isDefined)
```

Comment:
But the project actor can possibly be on another node, while distributed data cache should be distributed to each node, so in the multi node setup this could mean another network call

Reply:
I agree with @imsdu on this. The check needs to be consistent even if it would mean making a remote call.